validate.js

```diff
@@ -235,7 +235,7 @@ async function validate() {
   const results = {};
   results.errors = allerrors;
   results.timestamp = new Date();
-  results.repos = allrepos;
+  results.repos = allrepos.filter(r => !r.isPrivate);
```

Comment:
To save some work and not have the condition twice, can you instead filter in `async function sequenced`?

Reply:
@dontcallmedom did you mean to resolve this?

Reply:
doh, my commit had failed! fixed now